src/pre.l

```diff
@@ -957,6 +958,12 @@ WSopt [ \t\r]*
                                           yyextra->yyColNr+=(int)yyleng;
                                           BEGIN(DefName);
                                         }
+<Command>"cmakedefine01"{B}+            {
+                                          yyextra->potentialDefine += substitute(yytext,"cmakedefine01","     define");
```

Comment:
Is this substitution supposed to preserve the number of characters, as above? If so, it looks like it needs two more spaces.

Reply:
At first I thought it as well but I think that it is about where the `define` starts.

Reply:
I rethought about it and it is indeed better to replace `cmakedefine01` with `     define  ` although I didn't see any problems (tested `GENERATE_XML=YES`).